src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
-    this.callback({
-      srcEvent: event,
-      target: this.element
-    });
+    switch (event.type) {
+    case 'mousedown':
+      if (event.button === 0) {
+        this.pressed = true;
+      }
+      break;
+    case 'mousemove':
+      if (event.which !== 1) {
```

Comment:
Do `mousedown` and `mousemove` events name this member differently: `event.button` vs. `event.which`?  I think that is worth a comment, for a reviewer it looks like a typo.